### PR TITLE
Jj photogram ajax

### PR DESCRIPTION
**Pull request by jessjno**
refactored the likes/unlike and follow/unfollow buttons

### CODE REVIEW

File: app/views/follow_requests/_follow_unfollow.html.erb
Comment (merrillhuang) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: Good use of dom_id to assign unique id for element.

File: app/views/follow_requests/_form.html.erb
Comment (merrillhuang) [inline app/views/follow_requests/_form.html.erb:1]: Nice! I didn't think of using the recipient to get a unique identifier.

File: app/views/follow_requests/_follow_unfollow.html.erb
Comment (merrillhuang) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: Good use of dom_id to assign unique id for element.

**Review by merrillhuang — APPROVED** [review]
Everything works as expected. Nice job!

**Review by tluu5 — COMMENTED** [review]
Great job adding the format.js on def creat, update, and destroy

**Review by tluu5 — COMMENTED** [review]
catching the little mistakes

**Review by tluu5 — APPROVED** [review]
Looking good on writing the current_followers and updated_followers js

**Pull request merged**